docker-entrypoint.sh

```diff
@@ -365,6 +418,19 @@ docker_setup_db() {
 		fi
 	fi
 
+	# To create replica user
+	local createReplicaUser=
+	local changeMasterTo=
+	local startReplica=
+	if validate_replication ; then
+		changeMasterTo="CHANGE MASTER TO MASTER_HOST='$MARIADB_MASTER_HOST', MASTER_USER='$MARIADB_REPLICATION_USER', MASTER_PASSWORD='$MARIADB_REPLICATION_PASSWORD', MASTER_PORT=$MARIADB_MASTER_PORT, MASTER_CONNECT_RETRY=10;"
```

Comment:
At this line, we're only using `MARIADB_REPLICATION_PASSWORD` for replication not making use of `MARIADB_REPLICATION_PASSWORD_HASH`, So How could be replication started with only `MARIADB_REPLICATION_PASSWORD_HASH` without `MARIADB_REPLICATION_PASSWORD`, I don't see any document on mariadb for `change master to` using `MARIADB_REPLICATION_PASSWORD_HASH`

Reply:
`MARIADB_REPLICATION_PASSWORD_HASH` cannot be used on the replica mode because of this.

Reply:
Got it :)